CMakeLists.txt

```diff
@@ -147,6 +147,8 @@ else()
     PRIVATE
       triton-backend-utils    # from repo-backend
       triton-core-serverstub  # from repo-core
+      cuda
+      cudart
```

Comment:
Reminder: Remove this once we remove the temporary use of `cuMemGetAddressRange()`